templates/repo/issue/view_content/comments.tmpl

```diff
@@ -574,7 +574,6 @@
 				{{template "repo/commits_list_small" dict "comment" . "root" $}}
 			{{end}}
 		{{else if eq .Type 30}}
-			{{if not $.UnitProjectsGlobalDisabled}}
```

Comment:
Condition no longer needed?

Reply:
I do not think the condition makes sense. No matter the project is disabled or not, the issue activities should be displayed as-is.

If the condition would be considered as needed, please help to show a real case for it.

Reply:
Seems fine. I agree it does not make sense to hide it.